pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,29 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "mars_relief": GMTRemoteDataset(
+        title="Mars relief",
+        name="mars relief",
+        long_name="Mars relief",
```

Comment:
For the datasets of the Earth, underscores are used for `name`.
```suggestion
        title="Mars relief",
        name="mars_relief",
        long_name="Mars relief",
```